.woodpecker/docker.yml

```diff
@@ -0,0 +1,200 @@
+pipeline:
+  build-web:
+    image: node:16-alpine
+    commands:
+      - cd web/
+      - yarn install --frozen-lockfile
+      - yarn build
+
+  ###############
+  # S e r v e r #
+  ###############
+
+  # needed until https://github.com/woodpecker-ci/woodpecker/pull/635
+  build-server:
```

Comment:
We are currently missing `publish-server-dryrun`, `publish-server-alpine-dryrun`, `publish-agent-alpine-dryrun` and `publish-cli-alpine-dryrun`

Reply:
no only dryrun for server is missing ...

Reply:
... I'd like to fix this in #635 ...

Reply:
But we do not have a dryrun for `publish-agent-alpine` atm? 🤔

Reply:
... I'll add it now - will be changed later